.github/workflows/csound_builds.yml

```diff
@@ -101,6 +101,53 @@ jobs:
           name: iOS-builds
           path: iOS/csound-iOS-7.0.0 # or path/to/artifact
 
+  ios_build_ioscross:
+    name: iOS build (ioscross/vcpkg)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: true
+
+      - uses: lukka/get-cmake@v3.29.0
+
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - name: Bootstrap VCPKG
+        run: ./vcpkg/bootstrap-vcpkg.sh
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install build-essential liblzma-dev libxml2-dev libz-dev libbz2-dev cpio ninja-build
+
+      - name: Download ioscross
+        run: |
+          curl -L "https://drive.usercontent.google.com/download?id=1COyrQ7ypTTVBiI9dBIOvswIm1CWbNFa0&confirm=xxx" | tar xvz
```

Comment:
Not a great solution if this is outside of the Csound org. If the link goes dead or whatever other reason, then the build breaks. It's also a bit opaque, not sure what is being executed here unless we look inside.

Reply:
I do agree that this is not a great solution but was the suggested approach in the past for a previous pr.
The steps to build the tar are found in the [REAMDE](https://github.com/nonameentername/csound/blob/dcc5108cf9986141816a1621fd0b989f1973d493/platform/ioscross/README.md#build-github-targz).
To avoid linking to a file outside of the Csound org and admins, I updated my pr to instead use the location specified in the variable `vars.IOSCROSS_DOWNLOAD_URL`.  The Csound admins can chose to upload this to a desired location and the set the variable to enable the job.

Reply:
I suggested caching the build files to reduce the 2.5 hour build time but it wasn't ideal like here either. Maybe it would be better to create a docker image that contains the build tool binaries already. At least we can have a record of how it's built and version the images.

Reply:
I updated the pr to use a docker image instead of an external file.